content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -25,11 +25,11 @@ The following image shows two instances of Car prefab in a level. The Car prefab
 
 ![Level in Prefab Edit Mode in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/level-prefab-edit.png)
 
-Overrides applied to prefab instances are registered for that individual prefab instance and are stored in the parent prefab that contains the prefab instances.
+Overrides applied to prefab instances are registered for that individual prefab instance and are stored in the focused prefab that contains the prefab instances.
```

Comment:
I am not sure if we can still "focused prefab" here. I think "the prefab being edited" would be better.